index.html

```diff
@@ -4479,6 +4479,50 @@ <h2>HTTP</h2>
         <section class="informative">
           <h2>Credential Type-Specific Processing</h2>
 
+          <p>
+RDF [[?RDF11-CONCEPTS]] is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
+<i>entity&#x2192;attribute&#x2192;value</i>) relationship, which is referred to
+as a <a>claim</a> in this specification. JSON-LD [[?JSON-LD11]] is a technology that enables
+the expression of RDF using idiomatic JSON, enabling developers familiar with
+JSON to write applications that consume RDF as JSON. In general, subjects are
+expressed as JSON objects with each property and value of the subject as a JSON
+key and value, respectively. A special affordance is made to express an
+identifier of a subject, if necessary, using the <code>id</code> key in this
+specification. See
+<a data-cite="?JSON-LD11#relationship-to-rdf">Relationship of JSON-LD to RDF</a>
+for more details.
+          </p>
```

Comment:
I'm concerned that this is going to come across as word salad for many readers, who probably don't care about RDF (or have heard bad things about it, some of it completely justified). There are phrases here like: 

* "technology for modeling graphs of statements" -- not experienced by most CS students much less web developers.
* "enables the expression of RDF using idiomatic JSON" 
* "consume RDF as JSON" -- They can read the JSON-LD spec to learn more about the details there...

IOW, I'm concerned that the opening paragraph is going to not land well -- most readers are probably reading this section because they /don't want to care/ about all that stuff. They just want to issue a VC, they don't want to pull in a bunch of unnecessary RDF stuff... or be reminded of it (because, ideally, they don't have to think about the vast majority of the stuff under the hood... just follow the fairly straight-forward instructions in this section.

Reply:
I think this text provides some brief context (no pun intended) that has been missing in every iteration of the VC data model spec to date. What is says here fills gaps that have continued to come up in questions from various different readers over time. We should keep it.

> IOW, I'm concerned that the opening paragraph is going to not land well -- most readers are probably reading this section because they /don't want to care/ about all that stuff. 

This provides a quick informative summary of what those things are instead of leaving some readers quite confused about why these technologies are used. There are readers who don't care at all (who this text is not for) and there are readers who want to know more but aren't going to read all of the links to the various underlying specs. This text informatively explains (and I think pretty tersely):

RDF lets you model `subject->property->value` statements (oh, that's what "claims" are, so that's why RDF is used). JSON-LD lets you express RDF in JSON (oh, JSON-LD enables RDF to happen in JSON; I know JSON, but how does it look?). Each JSON object is a subject with JSON keys as the properties of the subject and JSON values as the associated values... and the `id` property holds the subject's identifier (oh, so these technologies provide a *standard*, consistent way to express "claims" in JSON, i.e., it is not just "anything goes JSON").

We could explicitly say how this approach is fully compositional (whereas "anything goes JSON" is not), but we can perhaps assume readers will get that -- and we are trying to keep it short.

> ...ideally, they don't have to think about the vast majority of the stuff under the hood...

I agree that it's good when people don't have to know this. However, we also keep not telling the people who do want to know in a concise way -- without forcing them to deep-dive into the other specs. I think people have been asking for this.

Reply:
I *do* think this paragraph feels out of place in the "Credential-Type specific processing" section of the spec. I'd recommend we move this to another subsection under JSON-LD called "Relationship to RDF" so people can read it there.